docs/source/contrib_guide_code.rst

```diff
@@ -0,0 +1,80 @@
+==================================
+Contributing to the Code
+==================================
+
+How can I help?
+---------------
+Individuals are welcome, and encouraged, to submit issues, enhancements, and
+new idea to the codebase. If you are a first-time contributor looking to get
+involved with Jupyter, you can use the following query in a Github search to
+find beginner-friendly issues to tackle across the Jupyter codebase.
+
+**is:issue is:open is:sprint-friendly user:jupyter**
+
+Once you've found an issue that you are eager to solve, you can use the guide
+below to get started. If you experience any issues while working on the issue,
+you can contact the mailing list or the appropriate Gitter channel.
+
+1. Fork the repository associated with the issue you are addressing and clone
+it to a local directory on your machine.
+
+2. ``cd`` into the directory and create a new branch using ``git checkout -b
+insert-branch-name-here``. Pick a branch name that gives some insight into
+what the issue you are fixing is. For example, if you are updating the text
+that is logged out by the program when a certain error happens you might 
+name your branch `update-error-text`.
+
+3. Reference the README of the repository you have forked and cloned for
```

Comment:
3. Refer to the repository's README and documentation for details on configuring your system for development.